src/main/target/WORMFC/target.h

```diff
@@ -0,0 +1,140 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define TARGET_BOARD_IDENTIFIER "RRF4"
+
+#define USBD_PRODUCT_STRING     "Worm FC"
+
+#define LED0_PIN    				PA15
+#define LED1_PIN					PC14
+
+#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
+
+#define BEEPER                  	PB14
+#define BEEPER_INVERTED
+
+#define INVERTER_PIN_UART6      	PB13 // PC0 used as inverter select GPIO
+#define INVERTER_PIN_UART3      	PB12 // PC0 used as inverter select GPIO
+
+// MPU6500 interrupt
+#define USE_EXTI
+#define MPU_INT_EXTI            	PC4
+#define USE_MPU_DATA_READY_SIGNAL
+//#define DEBUG_MPU_DATA_READY_INTERRUPT
+
+#define ICM20689_CS_PIN          PA4
+#define ICM20689_SPI_INSTANCE    SPI1
+
+#define MPU6500_CS_PIN          	PA4
+#define MPU6500_SPI_INSTANCE    	SPI1
+
+// ACC section -- start
+#define ACC
+
+#define USE_ACC_SPI_ICM20689
+#define ACC_ICM20689_ALIGN       CW180_DEG_FLIP
+
+#define USE_ACC_MPU6500
```

Comment:
Depending what you are using only one definition is needed as long you don't have board variants with SPI and I2C sensor. Also do you have board variants with different Gyro/ACC sensors? If not i would remove the redundant sensor driver.

Reply:
So I use only ICM20602 currently it’s under MPU6500 which is confusing, the problem is that it’s also under icm20689... then what driver should be used for ICM20602?? Also when you use SPI you have to include the I2C Version anyway since SPI is sort of overlay on top of I2C driver i think it’s quite confusing but whatever

Reply:
SInce both drivers support the ICM20206 you can chose one of it. Sure both MPU6500 drivers (SPI and I2C) need to be in the code since some functions are shared. Only the SPI defines are required to make it work properly for SPI.